### ISSUE TITLE
refmterr doesn't support OCaml 4.09.0

### ISSUE THREAD

**Issue by imbsky** [facebookexperimental/reason-native#221]

**Comment by jordwalke** [comment]
Thanks, how did you discover this issue? We need to fix this.

**Comment by mnxn** [comment]
I've noticed this too.
Simplest esy project to reproduce is just this `package.json`:
```json
{
  "name": "refmterr-ocaml49",
  "dependencies": {
    "ocaml": "4.9.0"
  },
  "devDependencies": {
    "refmterr": "3.2.2"
  }
}
```

Explanation is pretty simple, both the opam and esy file for refmterr have constraints to exclude ocaml 4.09.0:
https://github.com/facebookexperimental/reason-native/blob/master/refmterr.json#L22
https://github.com/facebookexperimental/reason-native/blob/master/refmterr.opam#L15

I built refmterr on my machine with ocaml 4.09.0 and it seemed to build with no problems. Should be a fairly simple fix to update the constraints.
